packages/contracts/README.md

```diff
@@ -16,17 +15,13 @@
 
 The Snickerdoodle Contracts stack consists of the following primary components:
 
-### [consent](/packages/contracts/contracts/consent/README.md)
+### [ad-wallet](/packages/contracts/contracts/user-wallet/README.md)
 
-Contains an upgradable EIP721 compatible NFT implementation and an associated contract factory.
+Contains ad wallet contracts
 
-### [token](/packages/contracts/contracts/token/README.md)
+### [user-wallet](/packages/contracts/contracts/user-wallet/README.md)
```

Comment:
I'm a little confused, we have the smart-wallet contract above; is that the name we're using, or are we sticking with user wallet?